mds/access_control/stateless_jwt.py

```diff
@@ -0,0 +1,31 @@
+from django.utils.encoding import smart_text
+from rest_framework.authentication import BaseAuthentication, get_authorization_header
+
+from mds.server.settings import AUTH_MEANS
+from .authenticate import authenticate
+
+
+class StatelessJwtAuthentication(BaseAuthentication):
```

Comment:
Why not subclass [TokenAuthentication](https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication) ?

Reply:
If i'm not mistaken, TokenAuthentication uses a Token & User stored in local database, we want neither